File: benchmarks/xlang/src/fib.js
function fib(n) { return n < 2 ? n : fib(n - 1) + fib(n - 2); }
const t0 = performance.now();
const r = fib(32);
console.log(`CHECK ${r}`);
console.log(`MS ${Math.round(performance.now() - t0)}`);

File: benchmarks/xlang/src/fib.R
fib <- function(n) if (n < 2) n else fib(n - 1) + fib(n - 2)
t0 <- proc.time()[["elapsed"]]
r <- fib(32)
ms <- as.integer((proc.time()[["elapsed"]] - t0) * 1000)
cat("CHECK", r, "\n")
cat("MS", ms, "\n")
